bundles/org.openhab.binding.livisismarthome/src/main/resources/OH-INF/i18n/livisismarthome.properties

```diff
@@ -3,24 +3,16 @@
 addon.livisismarthome.name = LIVISI SmartHome Binding
 addon.livisismarthome.description = The LIVISI SmartHome binding integrates your LIVISI SmartHome system.
 
-# common messages
-
-error.connect = Can't connect to LIVISI SmartHome service! Please check your credentials.
```

Comment:
The i18n plugin does not detect all placeholders. 
This is removed, but used here: https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.livisismarthome/src/main/java/org/openhab/binding/livisismarthome/internal/handler/LivisiBridgeHandler.java#L170

There are certainly more bindings that have similar issues.

Reply:
I have re-added them to the bottom.